### PR TITLE
Moves Smoke Machine boards to tech storage

### PR DESCRIPTION
**Pull request by Bawhoppen**
Seriously man?

### CODE REVIEW

**Review by Cyberboss — DISMISSED** [review]

**Comment by Robustin** [comment]
Seriously Man?

It's a perfect balance between convenience without spoon feeding. 

Put it in tech storage and it might as well be "The Smoke Machine that everyone but Chemists will be using".

**Comment by jughu** [comment]
lets lock something exclusive to chemistry that science will just raid roundstart because chemists couldnt get to it!

**Comment by BeeSting12** [comment]
Yea who the fuck wants to talk to other players to get what they want in a roleplaying game lol

**Comment by ShizCalev** [comment]
@Bawhoppen conflict

**Comment by DaxDupont** [comment]
After seeing Robustin borderline grief 5 rounds in a row I think this is a good idea.

**Comment by Robustin** [comment]
>Put out carpet in the halls

B O R D E R L I N E G R I E F

**Comment by ShizCalev** [comment]
deleted boxstation, gg

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 1]
```diff
@@ -58904,14 +58904,14 @@
...
- 	height = 24;
+	dheight = 9;
+	dir = 2;
+	dwidth = 5;
+	height = 24;
 	id = "syndicate_nw";
 	name = "northwest of station";
 	turf_type = /turf/open/space
-},
+	},
```
Comment (Bawhoppen) [inline _maps/map_files/BoxStation/BoxStation.dmm]: what the fuck

**Review by ShizCalev — APPROVED** [review]

**Pull request merged**